swarm/api/routes/runs_control.py

```
@@ -0,0 +1,795 @@
+
+        if state["status"] not in ("pending", "running"):
+            raise HTTPException(
+                status_code=409,
+                detail={
+                    "error": "invalid_state",
+                    "message": f"Cannot pause run with status '{state['status']}'",
+                    "details": {"current_status": state["status"]},
+                },
+            )
+
+        now = datetime.now(timezone.utc).isoformat()
```

Ruff (0.14.13)

Use `datetime.UTC` alias

Convert to `datetime.UTC` alias

(UP017)